app/javascript/src/components/Property/PropertyDetails.css

```diff
@@ -0,0 +1,20 @@
+.hackney-property-alerts {
+	width: 60%;
+	padding: 0;
+}
+  
+.hackney-property-alerts li {
+	  font-family: Arial, Helvetica, sans-serif !important;
+	  text-transform: capitalize;
+	  margin-bottom: 5px;
+	  padding: 4px 8px;
+	  font-weight: normal;
+}
+
+.bg-orange {
+	  background-color: #FFA300;
+}
```

Comment:
Since we're using SCSS you could extract the `#FFA300` into its own variable. maybe `$bg-orange: #FFA300;`